spanner_orm/model.py

```diff
@@ -111,55 +122,41 @@ def __init__(self, values, persisted=False):
           'All primary keys must be specified. Missing: {keys}'.format(
               keys=missing_keys))
 
-    self.start_values = {
-        key: copy.copy(value)
-        for key, value in values.items()
-        if key in self.columns() and value is not None
-    }
-    for name, value in self.start_values.items():
-      try:
-        self._validate(name, value)
-      except AssertionError as ex:
-        raise error.SpannerError(*ex.args)
+    self.start_values, self.related = {}, {}
+    for key, value in values.items():
+      if key in self.relations():
+        self.related[key] = value
```

Comment:
Is there any validation we could/should be doing on relations?

Reply:
if we ever do database writes as a result of setting relations, perhaps. since that is not currently the case, there's no need for validation right now